torch_xla/runtime.py

```diff
@@ -253,6 +253,15 @@ def use_spmd(auto: Optional[bool] = False):
     torch_xla._XLAC._xla_set_auto_sharding()
     os.environ["XLA_AUTO_SPMD"] = "1"
 
+  if runtime.device_type() == 'NEURON':
```

Comment:
it is the same file, I think you can just `device_type()`?

Reply:
Indeed, artifact from the other CR. Fixed it.